tests/test_observables.py

```diff
@@ -478,6 +509,16 @@ def test_mb_get(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_urlscan_hash(self, mock_get=None, mock_post=None):
```

Comment:
about tests, this a good step forward. But there are 2 problems:

1 - you have to test **both** the analyzers you created. In this way you are just testing the first analyzer. (also the analyzer name is wrong: "Urlscan" != "UrlScan_Search")

2 - "UrlScan_Submit_Result" supports only urls so its testing suite can be put here, in the URL section. Otherwise "UrlScan_Search" supports all the 4 types of observables and, like you did, you are duplicating the code. You should put the testing suite of this analyzer here https://github.com/intelowlproject/IntelOwl/blob/master/tests/utils.py#L25. In this way you just write the test one time but it will be applied for all the observables types

Reply:
Hey @mlodic , I have changed the tests. The file is able to pass the tests if I run the tests by passing the test parameters as a standalone application. I am still not able to debug why it's failing in the Travis build. Could you please help me with this. Sorry for bothering you!